### PR TITLE
test: increase branch coverage for get_form function in http/request/…

### PR DESCRIPTION
**Pull request by omidhzr**
…form.py, issue #16

### CODE REVIEW

File: tests/test_http_request.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 from scrapy.http import Request, FormRequest, XmlRpcRequest, JsonRequest, Headers, HtmlResponse
 # GROUP 12 ADDED IMPORT
-from scrapy.http.request.form import _get_inputs
+from scrapy.http.request.form import _get_inputs, get_form
```
Comment (majate) [inline tests/test_http_request.py]: typo: `get_form` &rightarrow; `_get_form`

File: tests/test_http_request.py [hunk 2]
```diff
@@ -19,6 +19,21 @@ def test_formdata_type(self):
         self.assertRaises(ValueError, _get_inputs, "", 5, "", "", "")
 
 
+class FormTest(unittest.TestCase):
+    # <input type="hidden" name="test" value="val1">
```
Comment (majate) [inline tests/test_http_request.py]: To keep the format of the tests written by @jenniferlindberg, "# GROUP 12 ADDED TEST CASE" should be added as comment to the implemented test. Potentially the current comment could be replaced?

**Review by majate — CHANGES_REQUESTED** [review]
Great test! Just a simple typo in the import that crashes the test. When that is fixed, the test runs as expected and increases the branch coverage 👌

File: tests/test_http_request.py (hunk 2 above)
Comment (omidhzr) [reply tests/test_http_request.py]: yeah, makes sense! thanks

File: tests/test_http_request.py (hunk 1 above)
Comment (omidhzr) [reply tests/test_http_request.py]: oops! I missed it in copy paste :)

**Review by jenniferlindberg — APPROVED** [review]
Great job Omid! Except for the things mentioned by @majate everything looks good :smiley: After fixing that I think that it is ready for merge!

**Review by majate — APPROVED** [review]

**Pull request merged**